mex/extractors/odk/transform.py

```diff
@@ -189,9 +189,8 @@ def transform_odk_data_to_extracted_variables(
         for resource in extracted_resources_odk
     }
     for file in odk_raw_data:
-        used_in = resource_id_by_identifier_in_primary_source[
-            file.file_name.split(".")[0]
-        ]
+        file_name = file.file_name.split(".")[0]
```

Comment:
```suggestion
        file_name = file.file_name.split(".xlsx")[0]
```

as an idea, to increase likelihood to just cut off the file extension, and no accidental "." in the filename